manifests/kustomize/base/cache-deployer/cluster/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - cache-deployer-clusterrole.yaml
+  - cache-deployer-clusterrolebinding.yaml
+  - cache-deployer-sa.yaml
```

Comment:
Service account is not cluster scoped.

But your clusterrolebinding will need to refer to this service account. I'm not sure what happens if they are in separate folders. Can you try if there are any problems?